tests/test_models.py

```diff
@@ -282,3 +283,20 @@ def test_dispatch_model_solve_case5(technology_set_4, net_demand):
     total_gen = model.results[['Nuclear',
                                'Curtailment']].sum().sum()
     assert (total_gen - net_demand.sum()) == pytest.approx(0, abs=TOL)
+
+
+def test_dispatch_model_solve_case6(technology_set_4, net_demand):
+    """
+    Tests that the reliability technology behaves as expected.
+    """
```

Comment:
I get the purpose of this test, but do you have a test in here to test  when `allow_blackout` is False? I see you have that in `test_dispatch_model_initialize` but not for any of the tests in which a solver is being used.

Reply:
Fair enough, I changed one of the tests to turn off the reliability function. I was mostly interested in making sure that adding this feature didn't change the results for the existing tests when it's included.